packages/devextreme/testing/tests/DevExpress.ui.widgets/treeViewParts/focusing.js

```diff
@@ -297,6 +298,20 @@ QUnit.test('select all item should be focused on treeview focus', function(asser
     assert.ok($selectAllItem.hasClass(FOCUSED_STATE_CLASS));
 });
 
+QUnit.test('Treeview container should be focused on empty treeview focus (T1294524)', function(assert) {
+    if(shouldSkipOnMobile(assert)) {
+        return;
+    }
+
+    initTree({
+        showCheckBoxesMode: 'selectAll',
+    }).dxTreeView('focus');
+
+    const $treeView = $(`.${TREEVIEW_CLASS_PREFIX}`);
```

Comment:
It is incorrect to find the element across the entire page. Use a search within the TreeView element instead.